### ISSUE TITLE
Context menu's more option is stuck in "... Loading"

### ISSUE THREAD

**Issue by piperun** [files-community/Files#13441]
### Description

Whenever I try and right-click to open the context menu,  the "More" option is not shown, and instead a greyed-out "...loading" will be shown.

What's worse is that this bug persists even if I close down Files completely.

However it does not appear to affect if you open up the context menu on a drive.

### Steps To Reproduce

Sadly I do not know exactly how this bug occurs.
1. Right-click on any folder or file
2. Context menu's option of "More" is replaced with a grey "...Loading" 

### Requirements

It should be able to load the "More" option or have a fail-safe to revert back to.

### Files Version

2.5.21.0

### Windows Version

10.0.19045.3393

### Log File

[debug.log](https://github.com/files-community/Files/files/12743809/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, can you see if this is an issue in the preview version. https://files.community/download

**Comment by yaira2** [comment]
@hishitetsu I believe this is fixed in `main`, can you confirm?

**Comment by hishitetsu** [comment]
My fix was merged before 2.5.29.0, so I guess this is another issue.

**Comment by piperun** [comment]
I can also add that restarting the computer makes the issue go away, but it will then reoccur at some point again.

**Comment by hishitetsu** [comment]
Perhaps it depends on your environment. Maybe one of the shell extensions is doing something bad.

**Comment by piperun** [comment]
If this were the case, is it possible to add a feature to alert the user of this?

**Comment by yaira2** [comment]
@piperun which shell extensions do you have in the context menu?

**Comment by piperun** [comment]
Quite a few:
![image](https://github.com/files-community/Files/assets/1145126/794f377b-7099-47f8-86bf-fe4ef1d43673)

**Comment by yaira2** [comment]
I don't think any of those should be causing issues

**Comment by hishitetsu** [comment]
> @hishitetsu is there any logging to check for a "bad" extension?

No, it just catches exceptions and ignores them without logging.

**Comment by Pietro228** [comment]
![obrazek](https://github.com/files-community/Files/assets/53868994/a65a1b23-04f2-47e2-b0f6-e13e855b8226)
Same for me. I use SVN Tortoise which works in right click so I really need it :/

**Comment by hishitetsu** [comment]
"Loading..." will disappear when an exception occurs during loading. So if "Loading..." remains, the loading process may be taking a very long time or get stuck.

**Comment by Pietro228** [comment]
Ať first it dosappears and then it just stays on "Loading..."

**Comment by Josh65-2201** [comment]
Is this still an issue in 3.7.11?

**Comment by Pietro228** [comment]
I haven't encountered this issue now.

**Comment by yaira2** [comment]
Thank you for the update.
